tests/virt_autotest/verify_virtualization.pm

```diff
@@ -37,6 +38,21 @@ sub run {
     return $self;
 }
 
+sub verify_bootloader {
+    my $self = shift;
+
+    my %osinfo = script_output("cat /etc/os-release") =~ /^([^#]\S+)="?([^"\r\n]+)"?$/gm;
+    %osinfo = map { uc($_) => $osinfo{$_} } keys %osinfo;
+    if (script_run("ls -d /sys/firmware/efi") != 0) {
+        record_info("Not uefi boot on system $osinfo{VERSION}", "Current system is running $osinfo{ID} $osinfo{VERSION} which does not use uefi boot");
+    }
+    else {
+        record_info("Uefi boot on system $osinfo{VERSION}", "Current system is running $osinfo{ID} $osinfo{VERSION} which boots with uefi firmware");
```

Comment:
same comment here, you don't see this text by design: https://openqa.suse.de/tests/19206773#step/verify_virtualization/28 it might help you to find a shorter version.

Reply:
oh, didn't know that, uff hover! hehe.... I don't like it much by design tbh, specially when there are more boxes, for the tester is not really useful for fast review, but all good, it is up to you specially in the files you team owns what is more convenient, thanks.